### PR TITLE
Adjustments for upstream functional CI

### PR DESCRIPTION
**Pull request by pleimer**
This PR will be used to track all adjustments required for the new CI system and remove code for legacy CI

### CODE REVIEW

File: .jenkins/agent/README.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+The Jenkins agent pod is used to run all Jenkins pipelines for the Service Telemetry Framework.
+
+# Build in OpenShift
+```bash
+oc new-build jenkins-agent --binary=true --to=jenkins-agent:latest
+oc start-build jenkins-agent --from-dir .
+```
+The will be available in-cluster at the address: `image-registry.openshift-image-registry.svc:5000/<NAMESPACE>/jenkins-agent:latest`
```
Comment (leifmadsen) [inline .jenkins/agent/README.md:8]:
```suggestion
Builds will be available in-cluster at the address: `image-registry.openshift-image-registry.svc:5000/<NAMESPACE>/jenkins-agent:latest`
```

**Review by leifmadsen — APPROVED** [review]

**Comment by pleimer** [comment]
test

**Comment by pleimer** [comment]
test

**Comment by pleimer** [comment]
test

**Comment by pleimer** [comment]
test

**Comment by pleimer** [comment]
test

**Comment by pleimer** [comment]
test

**Comment by pleimer** [comment]
test

**Comment by pleimer** [comment]
test

**Comment by pleimer** [comment]
test

**Comment by pleimer** [comment]
test

**Comment by pleimer** [comment]
test

**Comment by pleimer** [comment]
test

**Comment by pleimer** [comment]
test

**Comment by pleimer** [comment]
test

**Pull request merged**